stack-guideline.md

```diff
@@ -0,0 +1,65 @@
+# Abtion stack
+###### Or Ab-*stack*-tion!  🤦‍♂️
+![abtion image](https://media.giphy.com/media/ZVik7pBtu9dNS/giphy.gif)
+
+
+In Abtion we strive for the most efficient and easiest workflow.
+This docs is thought of being a docs we all can turn to if we get in doubt of how we deal with managing certain stuff, such as CI/CD.
+Also, to make it easier to on-board new employees.
+
+
+## Frontend
+As Abtioneers we love React when working on the FE (Front-End), but we also care about Vue and are always curious of what exciting technologies might pop-up in the future. But for now most of our FE apps are built with React.
+
+#### Testing
+React and [Jest](https://jestjs.io/) make a great match, which is why we always test our React apps with Jest.
+
+#### CI
+So far we are using both [Travis](https://travis-ci.org/) and [Buddy](https://buddy.works/) - not at the same time of course  💩. 
```

Comment:
And Github Actions, Peakflow, and Heroku CI.
For our newest project we landed onusing Heroku CI.
Though we do have an issue with it being too slow because the google-chrome buildpack for Heroku doesn't use caching.